packages/sanity/src/tasks/i18n/resources.ts

```diff
@@ -125,6 +125,10 @@ const tasksLocaleStrings = defineLocalesResources('tasks', {
   'form.status.success': 'Task created',
   /** The text displayed when no tasks are found */
   'list.empty.text': 'No tasks',
+  /** The text for the link displayed at the bottom of the tasks list inviting users provide feedback */
+  'list.feedback.link': 'share feedback on Tasks',
```

Comment:
Can't splitting the sentence like this "disturb" the translation and the sentence won't be correctly translated in all languages?